hack/update-deps.sh

```diff
@@ -22,3 +22,5 @@ rm -rf vendor
 dep ensure -v
 
 git apply hack/maistra-codegen.patch
+git apply hack/pkg-codegen.patch
```

Comment:
These are not codegen patches 🙂 . I'd name them something like `serving-listers.patch` or something like that. Maistra's patch literally fixes code generation.